src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -225,7 +233,9 @@ class SwipeableDrawer extends React.Component {
     this.removeBodyTouchListeners();
     this.setState({ maybeSwiping: false });
 
-    if (this.isSwiping === undefined) {
+    // The swipe wasn't started.
+    if (this.isSwiping !== 'opening' && this.isSwiping !== 'closing') {
+      this.isSwiping = undefined;
```

Comment:
if `isSwiping` is neither `opening` nor `closing`, then it already is `undefined` or `null`, which could be checked by a simple `if (this.isSwiping == null) return` here instead of this verbose logic.

Reply:
Better be explicit about the different values:
- undefined
- false
- opening
- closing

Reply:
We could do that in a comment.